src/components/EmptyContent/EmptyContent.vue

```diff
@@ -20,9 +20,17 @@
 -
 -->
 
+<docs>
+	### General description
+
+	This component provides a way to standardize the empty content.
+	The standard properties like empty text and description allow to automatically
+	include a standard empty content.
```

Comment:
With an example? :)

Reply:
How bad would it be to move on without it? :) i am doing mail stuff now, but can get back to this one on Monday

Reply:
I should not be that complicated to add ^^
Let me add one

Reply:
done

Reply:
Thank you <3 my laptop doesnt like when i open 2 different phpstorms, i need to close nc-vue in order to work on nc/mail, so was not a matter of how hard it would be.